files/SQLite.props

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition="!Exists('$(MSBuildProjectName).project.json') And !Exists('project.json') And '$(Platform)' == 'AnyCPU'">
+  <ItemGroup Condition="!Exists('$(MSBuildProjectName).project.json') And !Exists('project.json') And '$(Platform)' == 'AnyCPU' And '$(ExcludeSqliteNative)' != 'true' ">
```

Comment:
project.json will go away for .NET Core projects, it would be safer to check for packages.config and packages.{projName}.config here.